core/appeal/service.go

```diff
@@ -188,25 +188,56 @@ func (s *Service) Create(ctx context.Context, appeals []*domain.Appeal, opts ...
 		resourceIDs = append(resourceIDs, a.ResourceID)
 		accountIDs = append(accountIDs, a.AccountID)
 	}
-	resources, err := s.getResourcesMap(ctx, resourceIDs)
-	if err != nil {
-		return err
-	}
-	providers, err := s.getProvidersMap(ctx)
-	if err != nil {
-		return err
-	}
-	policies, err := s.getPoliciesMap(ctx)
-	if err != nil {
-		return err
-	}
 
-	pendingAppeals, err := s.getAppealsMap(ctx, &domain.ListAppealsFilter{
-		Statuses:   []string{domain.AppealStatusPending},
-		AccountIDs: accountIDs,
+	eg, egctx := errgroup.WithContext(ctx)
+	var (
+	    resources map[string]*domain.Resource
+	    providers map[string]map[string]*domain.Provider
+	    policies map[string]map[uint]*domain.Policy
+	    pendingAppeals map[string]map[string]map[string]*domain.Appeal
+	)
+
+	eg.Go(func() error {
+		resourcesData, err := s.getResourcesMap(egctx, resourceIDs)
+		if err != nil {
+			return err
```

Comment:
```suggestion
			return fmt.Errorf("error getting resource map: %w", err)
```

let's wrap the error for easier debugging?